apps/condo/domains/analytics/tasks/helpers/sendDailyStatistics.js

```diff
@@ -0,0 +1,529 @@
+            if (organizationData.tickets.isExpired > 0) {
+                this.#statistics.tickets.isExpired.byOrganizations.push({ name: organizationData.organization.name, count: organizationData.tickets.isExpired })
+            }
+            this.#statistics.tickets.withoutEmployee.total += organizationData.tickets.withoutEmployee
+            if (organizationData.tickets.withoutEmployee > 0) {
+                this.#statistics.tickets.withoutEmployee.byOrganizations.push({ name: organizationData.organization.name, count: organizationData.tickets.withoutEmployee })
+            }
+
+            this.#statistics.incidents.water.push(...organizationData.incidents.water)
+        }
+
+        return this.#statistics
+    }
+
+    /**
+     *
+     * @param organizationId
+     * @return {Promise<OrganizationData>}
+     */
+    async #getOrganizationData (organizationId) {
+        if (CACHE.has(organizationId)) {
+            this.#logger && this.#logger.info({
+                msg: `Data for organization "${organizationId}" was taken from the cache`,
+                taskId: this.#taskId,
+                data: { organizationId, userId: this.#userId, currentDate: this.#currentDate },
+            })
+            return CACHE.peek(organizationId)
+        }
+
+        const organization = await getByCondition('Organization', {
+            id: organizationId,
+            deletedAt: null,
+        })
+
+        if (!organization) {
+            throw new Error(`cannot find organization by id: "${organizationId}"`)
+        }
+
+        const {
+            withoutEmployee,
+            isExpired,
+            isReturned,
+            isEmergency,
+            inProgress,
+        } = await this.#getTicketsData(organizationId)
+        const { water } = await this.#getIncidentsData(organizationId)
+
+        const result = {
+            organization,
+            tickets: {
+                inProgress,
+                isEmergency,
+                isReturned,
+                isExpired,
+                withoutEmployee,
+            },
+            incidents: {
+                water,
+            },
+        }
+
+        CACHE.set(organizationId, result)
+
+        return result
+    }
+
+    /**
+     *
+     * @param organizationId
+     * @return {Promise<OrganizationTicketsData>}
+     */
+    async #getTicketsData (organizationId) {
+        const { count: inProgress } = await itemsQuery('Ticket', {
+            where: {
+                deletedAt: null,
+                organization: { id: organizationId, deletedAt: null },
+                status: { type: PROCESSING_STATUS_TYPE },
+            },
+        }, { meta: true })
+        const { count: isEmergency } = await itemsQuery('Ticket', {
+            where: {
+                deletedAt: null,
+                organization: { id: organizationId, deletedAt: null },
+                status: { type: NEW_OR_REOPENED_STATUS_TYPE },
```

Comment:
We check only tickets in open status?
Opened today or for all time?

Reply:
only tickets in open status for all time